### PR TITLE
Update rubygem-foreman_scc_manager to 2.2.0 Foreman 3.5

### PR DESCRIPTION
**Pull request by nadjaheitmann**
(cherry picked from commit d2a38afd5f86a545ab53f20378cdfb8d219653f8)

### CODE REVIEW

File: packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec [hunk 1]
```diff
@@ -113,6 +112,13 @@ cp -a .%{gem_dir}/* \
 %{gem_instdir}/test
 
 %changelog
+* Wed Jan 11 2023 Nadja Heitmann <nadjah@atix.de> 2.2.0-1
+- Update to 2.2.0
+- Fix permission issue for new installations
+
+* Mon Nov 14 2022 Bernhard Suttner <suttner@atix.de> 2.1.0-1
+- Update to 2.1.0
```
Comment (ekohl) [inline packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec:120]: This was a different commit. What we always do is cherry pick both commits so you get the correct log/references. You can feed `git cherry-pick -x` multiple commits to make this easier.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec (hunk 1 above)
Comment (nadjaheitmann) [reply packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec:120]: > This was a different commit. What we always do is cherry pick both commits so you get the correct log/references. You can feed `git cherry-pick -x` multiple commits to make this easier.

Thanks for the hint, changed the PR accordingly! Is it still possible to CP something to 'rpm/3.3 branch'?

**Review by ekohl — APPROVED** [review]
Thanks! Notice how this also included the proper BuildRequires update so they're in sync? That's precisely the reason we cherry pick as much as possible.

File: packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec (hunk 1 above)
Comment (ekohl) [reply packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec:120]: Yes. We still have the pipelines in place even though we officially ended support for 3.3.

**Comment by nadjaheitmann** [comment]
> Thanks! Notice how this also included the proper BuildRequires update so they're in sync? That's precisely the reason we cherry pick as much as possible.

Yeah, thanks, the feature is very handy, indeed - I like it!

**Pull request merged**